backend/dal/Migrations/v01.06.03/Down/PostDown/01-ViewProperties.sql

```diff
@@ -0,0 +1,219 @@
+PRINT 'Updating View_Properties'
```

Comment:
Make sure you and @goemen don't overwrite each other's work.  This one should just be a copy of the existing view before `v01.06.00`.

Reply:
Yeah, we'll see who ends up going first.